pkg/api/dtos/dashboard.go

```diff
@@ -7,28 +7,29 @@ import (
 )
 
 type DashboardMeta struct {
-	IsStarred   bool      `json:"isStarred,omitempty"`
-	IsHome      bool      `json:"isHome,omitempty"`
-	IsSnapshot  bool      `json:"isSnapshot,omitempty"`
-	Type        string    `json:"type,omitempty"`
-	CanSave     bool      `json:"canSave"`
-	CanEdit     bool      `json:"canEdit"`
-	CanAdmin    bool      `json:"canAdmin"`
-	CanStar     bool      `json:"canStar"`
-	Slug        string    `json:"slug"`
-	Url         string    `json:"url"`
-	Expires     time.Time `json:"expires"`
-	Created     time.Time `json:"created"`
-	Updated     time.Time `json:"updated"`
-	UpdatedBy   string    `json:"updatedBy"`
-	CreatedBy   string    `json:"createdBy"`
-	Version     int       `json:"version"`
-	HasAcl      bool      `json:"hasAcl"`
-	IsFolder    bool      `json:"isFolder"`
-	FolderId    int64     `json:"folderId"`
-	FolderTitle string    `json:"folderTitle"`
-	FolderUrl   string    `json:"folderUrl"`
-	Provisioned bool      `json:"provisioned"`
+	IsStarred            bool      `json:"isStarred,omitempty"`
+	IsHome               bool      `json:"isHome,omitempty"`
+	IsSnapshot           bool      `json:"isSnapshot,omitempty"`
+	Type                 string    `json:"type,omitempty"`
+	CanSave              bool      `json:"canSave"`
+	CanEdit              bool      `json:"canEdit"`
+	CanAdmin             bool      `json:"canAdmin"`
+	CanStar              bool      `json:"canStar"`
+	Slug                 string    `json:"slug"`
+	Url                  string    `json:"url"`
+	Expires              time.Time `json:"expires"`
+	Created              time.Time `json:"created"`
+	Updated              time.Time `json:"updated"`
+	UpdatedBy            string    `json:"updatedBy"`
+	CreatedBy            string    `json:"createdBy"`
+	Version              int       `json:"version"`
+	HasAcl               bool      `json:"hasAcl"`
+	IsFolder             bool      `json:"isFolder"`
+	FolderId             int64     `json:"folderId"`
+	FolderTitle          string    `json:"folderTitle"`
+	FolderUrl            string    `json:"folderUrl"`
+	Provisioned          bool      `json:"provisioned"`
+	ProvisioningFilePath string    `json:"provisioningFilePath"`
```

Comment:
Not a big fan of this name, not sure why but  think it's the mixing of "Provisioned" (For the bool prop) and "Provisioning"  prefix feels inconsistent.
Alt names
* ProvisionedSourceFile
* ProvisionedFilePath
* ProvisionedExternalId 

This is nitpicking but it's part of the public api and hard to change later. 

thoughts @bergquist  ?

Reply:
True did not give that much thought. I would go with `ProvisionedFilePath`. `ProvisionedExternalId` could be confusing as it is different than what is in external_id column in DB. For `ProvisionedSourceFile` I probably like to have the Path in the name as this could seem like some binary data or content string maybe.

Reply:
Ok makes sense in that case, will change it.